tests/util/DummyClam.php

```diff
@@ -65,7 +65,7 @@ protected function handleConnection($connection){
 
 			$dataChunk = '';
 			do {
-				$dataChunk .= @fread($connection, $chunkSize);
+				$dataChunk .= @fread($connection, $chunkSize - strlen($dataChunk) );
```

Comment:
after all the debugging - this was the thing to be taken care of. Needed some thinking of what is going on